Templates/bootstrap4_template.html

```diff
@@ -0,0 +1,218 @@
+                
+                {ifdef function}
+                <li class="nav-item">
+                    <a class="nav-link" href="../../index.html">API Home</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="../../contents.html">API Index</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link" href="index.html">Module</a>
+                </li>
+                {endif function}
+            </ul>
+            <div class="form-inline my-2 my-lg-0">
+                <button class="btn btn-outline-success my-2 my-sm-0" type="button">
+                    {subtitle}
+                    {ifdef home}{title}{endif home}
+                    {ifdef contents}{title}{endif contents}
+                </button>
+                <!--<input class="form-control mr-sm-2" type="text" placeholder="Search">
+                    <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>-->
+            </div>
+        </div>
+    </nav>
+
+    <div class="container-fluid">
+        <div class="var-breadcrumbs">{breadcrumbs}</div>
+
+        <!--<h2>{subtitle}</h2>-->
+        <h1 class="display-3">{title}</h1>
+
+        <!-- {preamble} = manifest.Description property + module_overview.html <body> contents -->
+
+        {ifdef module}
+        <p class="lead">{module.Description}</p>
+        {endif module}
+
+        <div class="var-body">{body}</div>
+
+        <hr>
+
+        {ifdef module}
+        <blockquote class="blockquote">
+            <dl class="row">
+                <dt class="col-sm-3">Manifest Name:</dt>
+                <dd class="col-sm-9">{module.Name}</dd>
+
+                <dt class="col-sm-3">Module Type:</dt>
+                <dd class="col-sm-9">{module.ModuleType}</dd>
+                
+                <dt class="col-sm-3">GUID:</dt>
+                <dd class="col-sm-9">{module.Guid}</dd>
+                
+                <dt class="col-sm-3">Version:</dt>
+                <dd class="col-sm-9">{module.Version}</dd>
+                
+                <dt class="col-sm-3">Author:</dt>
+                <dd class="col-sm-9">{module.Author}</dd>
+                
+                <dt class="col-sm-3">Company:</dt>
+                <dd class="col-sm-9">{module.CompanyName}</dd>
+                
+                <dt class="col-sm-3">Manifest Copyright:</dt>
+                <dd class="col-sm-9">{module.Copyright}</dd>
+                
+                <dt class="col-sm-3">Prefix:</dt>
+                <dd class="col-sm-9">{module.Prefix}</dd>
+                
+                <dt class="col-sm-3">Manifest Description:</dt>
+                <dd class="col-sm-9">{module.Description}</dd>
+            </dl>
+        </blockquote>
+        {endif module}
+
+        <div class="var-copyright">{copyright}</div>
+        <div class="var-revdate">{revdate}</div>
+
+    </div>
+    <!-- /.container -->
```

Comment:
The comment was corrected to match the corresponding class attribute of the opening DIV tag.